harness/mock-search/tool.py

```diff
@@ -26,14 +29,17 @@ def find(haystack, needle):
             break
 
 
-with open(argv[1], 'r') as haystack_file:
-    haystacks = load(haystack_file)
+if __name__ == '__main__':
+
+    run(['tar', 'xf', argv[1]])
+    run(['tar', 'xf', argv[2]])
 
-with open(argv[2], 'r') as needle_file:
-    needles = load(needle_file)
+    with open('query.fa', 'r') as needle_file:
+        needles = load(needle_file)
 
+    with open('targets.fa', 'r') as haystack_file:
+        haystacks = load(haystack_file)
```

Comment:
We don't want to hardcode the file names.

Reply:
Nevermind, I see why you did that. It's fine since you're passing in the tarball names.

Reply:
Not sure why this isn't shared.fa - i changed it in tool.py